metadata-ingestion/tests/integration/sigma/golden_test_platform_instance_ingest.json

```diff
@@ -545,6 +541,38 @@
         "lastRunId": "no-run-id-provided"
     }
 },
+{
+    "entityType": "dashboard",
+    "entityUrn": "urn:li:dashboard:(sigma,cloud_instance.OSnGLBzL1i)",
+    "changeType": "UPSERT",
+    "aspectName": "browsePathsV2",
+    "aspect": {
+        "json": {
+            "path": [
+                {
+                    "id": "urn:li:dataPlatformInstance:(urn:li:dataPlatform:sigma,cloud_instance)",
+                    "urn": "urn:li:dataPlatformInstance:(urn:li:dataPlatform:sigma,cloud_instance)"
+                },
+                {
+                    "id": "urn:li:dataPlatformInstance:(urn:li:dataPlatform:sigma,cloud_instance)",
+                    "urn": "urn:li:dataPlatformInstance:(urn:li:dataPlatform:sigma,cloud_instance)"
+                },
+                {
```

Comment:
We don't need to set platform_instance as part of browse path v2 . Its always set via auto_browse_path_v2_helper.

Reply:
Remove the code of adding platform_instance for browse path v2